renku_notebooks/api/classes/user.py

```diff
@@ -37,6 +39,14 @@ def jss(self):
         )
         return jss["items"]
 
+    @property
+    def datasets(self):
```

Comment:
Do we rely on the datasets being a user property? Would it be cleaner to treat them as a property of the user server?

Reply:
This is not really used. I will remove it. This is already a property of the user server.